tests/flamegraph.rs

```diff
@@ -76,30 +76,33 @@ where
     R: BufRead,
     E: BufRead,
 {
+    const BLESS_MSG: &str =
+        "If you want to bless the output, set `INFERNO_BLESS_TESTS=1` before the `cargo test`";
```

Comment:
I think "bless the output" isn't going to be super obvious to most people. How about:

> If you have modified the code and specifically want to update the expected test output, set  `INFERNO_BLESS_TESTS=1` before the `cargo test`.

Reply:
I used the same terminology as `rust-lang/rust` but you're right it isn't really understandable for someone new. Changed to your proposition.